lib/puppet/provider/network_interface/nxapi.rb

```diff
@@ -0,0 +1,82 @@
+# The NXAPI provider for network_interface
+#
+# October, 2015
+#
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Puppet::Type.type(:network_interface).provide(:nxapi, parent: Puppet::Type.type(:cisco_interface).provider(:nxapi)) do
+  @doc = 'network INTERFACE'
+
+  has_features :activable, :describable
```

Comment:
Ok.. I found :describable but it's under network_vlan? 

https://github.com/puppetlabs/netdev_stdlib/blob/f7184a30bed58635fc7e1a226f0a64530186ab5d/lib/puppet/type/network_vlan.rb#L9

I could not find :activable.